srcpkgs/void-docs/template

```diff
@@ -0,0 +1,18 @@
+# Template file for 'void-docs'
+pkgname=void-docs
+version=2020.04.18
+revision=1
+_commit=029fd1683100ae36c87e29227af8de4cadd1095b
+wrksrc="${pkgname}-${_commit}"
+build_style=meta
+short_desc="Documentation for Void Linux"
+maintainer="Érico Nogueira <ericonr@disroot.com>"
+license="CC-BY-SA-4.0"
+homepage="https://docs.voidlinux.org/"
+distfiles="https://github.com/void-linux/void-docs/archive/${_commit}.tar.gz"
+checksum=845e56b88bfa4a27ff42705051b319a92c96b02113122237ede58282f12648e8
+
+do_install() {
+	vmkdir usr/share/void-docs
+	vcopy src/* usr/share/void-docs
```

Comment:
Typically we avoid trying to package "bundles of files". See the "Quality Requirements" section of the manual for more information. One way to meet those requirements is to simply build the book with mdbook.

Reply:
Indeed, people using the flavored live images could enjoy proper visualization with html files. Do you think it would make sense to create separate folders for raw markdown and for html? Something like

* `/usr/share/void-docs/markdown`
* `/usr/share/void-docs/website`
* `/usr/share/void-docs/mandoc` -> this one is optional, but I used pandoc to turn the current markdown stuff into roff files and they look reasonably good.

Reply:
`html` might make more sense than `website`, or at least be more consistent with other packages.

Reply:
Fair enough.